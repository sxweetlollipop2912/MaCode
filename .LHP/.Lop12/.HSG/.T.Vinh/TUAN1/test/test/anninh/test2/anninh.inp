5 10
0 1
0 9
1 3
4 10
9 10
2 7
10 9
1 9
2 4
10 4
6 4
0 1
6 3
3 3
7 10
