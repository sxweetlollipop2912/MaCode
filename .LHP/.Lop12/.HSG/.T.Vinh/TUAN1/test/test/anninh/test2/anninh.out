2
1
1
1
6
5
0
5
2
2
